faceted_search/src/scripts/localdev.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo "In $PWD"
+cd node_modules/solr-faceted-search-react/
+echo "Starting watcher in solr-faceted-search-react"
+npm run watch &
+echo "Started watcher in solr-faceted-search-react"
+cd ../..
+echo "Starting local dev server"
+npm run start
```

Comment:
I thought I had checked this in before, but apparently I missed it.  Unrelated to this PR specifically.